lib/src/util/posix/dynamic_library.cc

```diff
@@ -41,7 +41,11 @@ namespace facter { namespace util {
         close();
 
         // Don't actually perform a load to determine if it is already loaded
+#ifdef _AIX
+        _handle = dlopen(name.c_str(), RTLD_LAZY);
```

Comment:
Absolutely. The status of this hack is the main reason this is marked do
not merge.

I'll move it to its own commit and comment that its a dirty hack
On May 22, 2015 3:55 PM, "Michael Smith" <notifications@github.com> wrote:

> In lib/src/util/posix/dynamic_library.cc
> <https://github.com/puppetlabs/facter/pull/953#discussion_r30936609>:
>
> > @@ -41,7 +41,11 @@ namespace facter { namespace util {
> >          close();
> >
> >          // Don't actually perform a load to determine if it is already loaded
> > +#ifdef _AIX
> > +        _handle = dlopen(name.c_str(), RTLD_LAZY);
>
> Sounds fine to me; I'd probably prefer a comment explicitly linking it to
> FACT-891.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/facter/pull/953/files#r30936609>.
>